examples/BuildOutOfContainer/BuildOutOfContainer.csproj

```diff
@@ -0,0 +1,16 @@
+<Project Sdk="Microsoft.NET.Sdk.Web">
+
+  <PropertyGroup>
+    <TargetFramework>net5.0</TargetFramework>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Include="Microsoft.ApplicationInsights.AspNetCore" Version="2.17.0" />
+    <PackageReference Include="Swashbuckle.AspNetCore" Version="5.6.3" />
+  </ItemGroup>
+
+  <ItemGroup>
+    <ProjectReference Include="..\..\src\ApplicationInsights.Kubernetes\ApplicationInsights.Kubernetes.csproj" />
```

Comment:
Consider forward slash if you want the project to be usable on Mac/Linux

Reply:
Having a little bit faith in `csc` :-) It understands the path for all OSes :-) Verified it on Ubuntu. Probably won't fix this. Either unfair to Windows Users or unfair to Unix users :-)